src/App.css

```diff
@@ -1,39 +1,57 @@
-.App {
-  text-align: center;
+* {
```

Comment:
**[OPTIONAL]**

- [ ] It would be nice if you use the same picture on your Twitter account which you have used in the LinkedIn because many recruiters are on twitter and when they will visit your twitter account then they should get a professional vibe from your account and the first thing they are gonna notice is your twitter account dp :thinking: so now you knows what you should do :+1:

Reply:
@HAMMAS-SALEEM  thanks for your Tips!!, I'll check it 👍